freqtrade/rpc/rpc.py

```diff
@@ -282,6 +284,10 @@ def _rpc_trade_statistics(
                 profit_ratio = trade.close_profit
                 profit_closed_coin.append(trade.close_profit_abs)
                 profit_closed_ratio.append(profit_ratio)
+                if trade.close_profit > 0:
```

Comment:
Kinda nitpicking, but I'd say `>= 0`. I wouldn't count a draw as a loss. It's neither of them, but you lost nothing.

Reply:
Well it's a discussion point for sure.
I've been thinking about this too - but in the end, a draw is not what you (should) aim for - it's profit - so this would mean that the strategy didn't do what it should've.

Now realistically it's almost impossible to reach "0 profit" - not after exchange rounding of both prices, amounts, fees ... you can be close to 0, like 0.00001 - or -0.00001 - but never 0...